packages/core/src/index.ts

```diff
@@ -4,7 +4,11 @@ export {
   fromEventObservable,
   fromObservable,
   fromPromise,
-  fromTransition
+  fromTransition,
+  type CallbackActorLogic,
+  type ObservableActorLogic,
+  type PromiseActorLogic,
+  type TransitionActorLogic
 } from './actors/index.ts';
```

Comment:
We can collapse this whole export into `export * from './actors/index.ts'`. Since both of those are publicly available~ (`xstate` and `xstate/actors`) and we export those `from*` factories from both, I think it's best to just always reexport here whatever is exported by that nested entrypoint

Reply:
Sounds good